src/Page.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Input from './Input';
+import List from './List';
```

Comment:
```suggestion
import React from 'react';

import Input from './Input';
import List from './List';
```

의미 상으로 구분되는 경우 빈 줄을 넣어서 구분지어주시면 코드 가독성을 높이는데 더 좋아요 ;)